foundation/foundation-upgrade/src/main/resources/migrations.json

```diff
@@ -91,6 +91,15 @@
           }
         ]
       },
+      {
+        "name": "data-lineage-package-import-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.v1_7_0.DataLineageAiopsReferencesMigration",
+        "fileSets": [
+          {
+            "includes": ["*-pipelines/*/src/*/java/*/cdi/*.java"]
```

Comment:
C: We need to migrate all Java files not just those in `cdi` packages of pipelines.  A project could easily be referencing this package through some custom lineage consumer.  Recommend changing this to `src/**/*.java`.

Reply:
You don't know that this is the only location though.  It's very likely that a project using data lineage would also add their own custom code that leverages the data lineage package.  The breakdown of `shouldExecuteOnFile` vs `performMigration` should give you what you need to address any performance issues.